test/e2e-techpreview/onclusterlayering_test.go

```diff
@@ -1053,3 +1053,76 @@ func TestSSHKeyAndPasswordForOSBuilder(t *testing.T) {
 		t.Logf("Deleted MachineConfig %s", testConfig.Name)
 	})
 }
+
+func TestExtensionsForOSBuilder(t *testing.T) {
```

Comment:
suggestion: Maybe this test could be combined with `TestOnClusterLayeringRollsOutImage`? I make this distinction because the tests where we build and apply an image and then roll back take a bit of time to execute and it would be a good idea to have the test do double-duty in this case.

Reply:
Fixed